cli/index.js

```diff
@@ -19,18 +18,23 @@ const msgBox = boxen(greeting, boxenOptions);
 
 // Cli commands implementation
 // TODO: move to a different folder so it looks more tidy
+
+// ==> authentication <==
+
+//*signup
 const signUp = (args) => {
 
-    const { email, password, username } = args;
+    const { email, password, username,Birthday } = args;
 
     const newUserData = {
         email,
         password,
         confirmPassword: password,
         username,
+        Birthday,
     };

```

Comment:
Again, why is birthday in uppercase and everything else in lowercase?

Check the contract. There are many parameters that are missing from this signUp.